Frequently-Asked-Questions.mediawiki

```diff
@@ -271,7 +270,7 @@ A Dutch step by step tutorial can be found [https://www.duken.nl/forums/handleid
 
 See [https://web.archive.org/web/20090320211618/http://btfaq.com:80/serve/cache/23.html this page]  for definitions of BitTorrent terms.
 
-== Hotkeys ==
+== What hotkeys/keyboard shortcuts are available? ==
```

Comment:
This list is incomplete IIRC.

Reply:
Fair -- could we get this version out and then create an issue to fill out the list?  If those are already documented somewhere, happy to pull them in, but if it requires code-discovery, I'd like to punt that effort if possible.

Reply:
You can change the description as you see fit if you want

Ctrl+F = Toggle focus between "Filter torrents..." and "Filter files..." also applies in search engine and new torrent dialog
Ctrl+E = same as above except for new torrent dialog

When focused in:
Peer list:
Ctrl+C = Copy all selected peers in this form "IP:PORT" one per line
Del = Ban peer

HTTP sources:
Del = Delete selected URL sources
F2 = Edit selected URL sources

RSS:
F2 = Edit selected rule/feed
Del = Delete selected rule/feed

Content:
Enter/Return = Open file/folder
F2 = Rename file/folder

Trackers:
F2 = Edit tracker (only the last one during multi-selection)
Del = Delete selected trackers
Ctrl+C = Copy selected trackers URLs

Transfer list:
Shift+Del = Delete torrent+files
Enter/Return = Double click action
Ctrl+M = Force start selected torrents (looks like Mac uses the same hotkey for minimize)

Everywhere:
F1 (or Ctrl+? for Mac) = Open the wiki page
Ctrl+i = Open statistics dialog
Alt+1 = Transfer list tab
Alt+2 = Search engine tab
Alt+3 = RSS tab

Bonus:
Ctrl + B = browse to a location in some fields (wasn't everywhere some time ago, not sure now)